src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -502,8 +498,8 @@ private async Task<IConfidentialClientApplication> BuildConfidentialClientApplic
         /// the client can trigger an interaction with the user so that the user consents to more scopes.
         /// </summary>
         /// <param name="scopes">Scopes to consent to.</param>
-        /// <param name="msalServiceException"><see cref="MsalUiRequiredException"/> triggering the challenge.</param>
-        public void ReplyForbiddenWithWwwAuthenticateHeader(IEnumerable<string> scopes, MsalUiRequiredException msalServiceException)
+        /// <param name="msalServiceException">The <see cref="MsalUiRequiredException"/> that triggered the challenge.</param>
+        public async Task ReplyForbiddenWithWwwAuthenticateHeader(IEnumerable<string> scopes, MsalUiRequiredException msalServiceException)
         {
```

Comment:
method name should end in `Async` if it's async now.

Reply:
Good catch. Updated.